vignettes/geologic-map.Rmd

```diff
@@ -39,46 +39,46 @@ install.packages("devtools")
 
 ```{r eval = TRUE}
 citation("rmacrostrat")
 ```
 
 # Context
-The Hell Creek is a geologic formation from the late Cretaceous and early Paleogene of North America, which is  found cropping out across Montana, North and South Dakota, and Wyoming in the United States. It consists of both freshwater and brackish clays, mudstones, and sandstones, representing a fluvial system on the shoreline of the Western Interior Seaway. The Hell Creek is famous for its preservation of an extensive array of both terrestrial and marginal marine fossils from just prior to the K/Pg mass extinction, including widely recognisable dinosaurs such as *Edmontosaurus*, *Triceratops* and *Tyrannosaurus rex*. In this vignette, we will show you how to use `rmacrostrat` to find the outcrop of the Hell Creek Formation, and use this to create a map which could be used in a potential publication. Whilst we have chosen a specific formation, this methodology could be applied to any number of geologic units of interest. So let's dive in!
+The Hell Creek is a geologic formation from the late Cretaceous and early Paleogene of North America, which is found cropping out across Montana and North and South Dakota, in the United States. It consists of both freshwater and brackish clays, mudstones, and sandstones, deposited by a fluvial system on the shoreline of the Western Interior Seaway. The Hell Creek is famous for its preservation of an extensive array of both terrestrial and marginal marine fossils from just prior to the K/Pg mass extinction (66 million years ago), including widely recognizable dinosaurs such as *Edmontosaurus*, *Triceratops* and *Tyrannosaurus rex*. In this vignette, we will show you how to use `rmacrostrat` to find the outcrop of the Hell Creek Formation, and use this to create a map which could be used in a potential publication. Whilst we have chosen a specific formation, this approach could be applied to any number of geologic units of interest. So, let's dive in!
 
 # Fetching
-First, we want to find the `strat_name_id`s that are associated with the Hell Creek, so we can grab all the available outcrop for the formation. We can do this by using some of the `def_` functions. These functions provide information about the information stored in the Macrostrat database. Let's have a look at the information associated with the Hell Creek Formation, using `def_strat_names()`:
+First, we want to find the `strat_name_id`s that are associated with the Hell Creek. These IDs are linked to unique stratigraphic names given to one or more geologic entities in the Macrostrat database. We can search for the relevant IDs by using some of the `def_*` functions. These functions provide information about the data stored in the database. Let's have a look at the information associated with the name Hell Creek, using `def_strat_names()`:
```

Comment:
I added a description of the IDs here but maybe shouldn't have used 'entities' here as this word is used below to describe the stratigraphic concepts. Not sure what would be a better way to phrase this?

Reply:
I've used 'units' here instead, and added reference to either Macrostrat Units or Map units. Hope that's alright.